constructor/_schema.py

```diff
@@ -692,16 +692,16 @@ class ConstructorConfiguration(BaseModel):
     """
     initialize_by_default: bool | None = None
     """
-    Default value for the option added by `initialize_conda`. The default
-    is true for GUI installers (EXE, PKG) and false for shell installers. The user
+    Default value for the option added by `initialize_conda`. The default is
+    true for PKG installers, and false for EXE (GUI and CLI installations) and shell installers. The user
     is able to change the default during interactive installation. NOTE: For Windows,
     `AddToPath` is disabled when `InstallationType=AllUsers`.
```

Comment:
```suggestion
    Default value for the option added by `initialize_conda`. The default is
    true for PKG installers, and false for EXE and SH shell installers.
    The user is able to change the default during interactive installations.
    Non-interactive installations are not affected by this value: users must explicitly request
    to add to `PATH` via CLI options.
    NOTE: For Windows, `/AddToPath` is disabled when `/InstallationType=AllUsers`.
```

I tried to make this explicit. Let me know what you think.